estate/models/estate_property.py

```diff
@@ -1,18 +1,45 @@
+
+#python import
 from email.policy import default
+from dateutil.relativedelta import relativedelta
 
+#odoo import
 from odoo import models, fields
 
+
+def _date_in_three_months(self:None):
+    return fields.Date.today() + relativedelta(months=3)
+
+
 class EstateProperty(models.Model):
     _name = "estate.property"
     _description = "This model is for your real estate company"
     
-    name = fields.Char(string="Name", required=True, default="House")
+    name = fields.Char(string="Name", required=True, default="House x")
+    # this info is useless to get displayed in form view
```

Comment:
Je ne suis pas d'accord avec ce commentaire, savoir si une popriété est archivée ou non est intéressante selon moi. On peut avoir un avis différent sur la question mais je recommande d'éviter de partager des opinions subjectives dans le code, uniquement des faits

Reply:
ahh oui j'ai mal ecrit mes comments , je voulais juste dire pluto "field" => "this field is useless to get displayed in form view" that why